fluss-dist/src/main/assemblies/plugins.xml

```diff
@@ -27,10 +27,11 @@
 
     <files>
         <!-- filesystem -->
+        <!-- output directory and plugin JAR should correspond to the file system *schema*, i.e., plugins/<schema>/ and fluss-fs-<schema>-${project.version}.jar -->
         <file>
             <source>../fluss-filesystems/fluss-fs-hadoop/target/fluss-fs-hadoop-${project.version}-with-dependencies.jar</source>
-            <outputDirectory>plugins/hadoop/</outputDirectory>
-            <destName>fluss-fs-hadoop-${project.version}.jar</destName>
+            <outputDirectory>plugins/hdfs/</outputDirectory>
+            <destName>fluss-fs-hdfs-${project.version}.jar</destName>
```

Comment:
Do we need to rename the jar as well? @luoyuxia

Reply:
It works whether rename or not rename the jar.  But I'd prefer not to rename it sot that we can know the jar is from the jar `fluss-fs-hadoop` built by our fluss-fs-hadoop moudle.

Reply:
+1